comment/locale/hi/LC_MESSAGES/django.po

```diff
@@ -93,7 +96,9 @@ msgstr "{model_id} मॉडल के लिए मान्य मॉडल 
 msgid ""
 "{parent_id} is NOT a valid id for a parent comment or the parent comment "
 "does NOT belong to the provided model object"
-msgstr "{parent_id} मूल टिप्पणी के लिए मान्य आईडी नहीं है या मूल टिप्पणी प्रदान की गई मॉडल ऑब्जेक्ट से संबंधित नहीं है"
+msgstr ""
+"{parent_id} मूल टिप्पणी के लिए मान्य आईडी नहीं है या मूल टिप्पणी प्रदान की गई मॉडल "
+"ऑब्जेक्ट से संबंधित नहीं है"
```

Comment:
has there been a change around the length after which strings are wrapped in a line? I can't seem to see any other change here.